install/kots/README.md

```diff
@@ -20,12 +20,15 @@ commercially supported project.
 
 # Development
 
+> tl;dr the `make` command will build and push a release to your development channel
+
 ## Authentication
 
-Two environment variables are required to be able to publish to our Replicated account:
+The following environment variables are required to be able to publish to our Replicated account:
 
  - `REPLICATED_APP`: the unique application slug
```

Comment:
We still use `gitpod-pov`, right? Eventually, we will use `gitpod` for prod but still keep a different dev app slug, right? Should we mention the app slugs in use here?

By the way: What do you think of renaming `gitpod-pov` to `gitpod-dev` (or create if renaming does not work)?

Reply:
Yes, we are using `gitpod-pov` which is what's defined by the `REPLICATED_APP` value. I'm not against documenting in there, but I don't think it's necessary

I can see why `gitpod-dev` would be preferable, but I'm not that bothered by renaming it. If the majority want to rename it, that's fine, but it's effectively our internal dev service so it's fine as is.

I think we'll likely abandon `gitpod-pov` in future and just use `gitpod` but that's a future discussion

Reply:
> I'm not against documenting in there, but I don't think it's necessary

How does a new colleague know which value to set here? Maybee, we need also a link to the vendor portal where an API key can be generated. Would make the onboarding of new folks easier when we have this written down somewhere and why not here in the readme?

Reply:
> Ok. I've changed the wording slightly.
> 
> The vendor portal is linked in the `REPLICATED_API_TOKEN`. I'm reluctant to duplicate too much of their development readme in ours as it means we then have to maintain that as well as our own application.

Awesome! Thanks a lot! :+1: